doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -270,149 +255,148 @@ automatically created.
    from the order in which they were inserted, making this implementation
    observationally different from the legacy one.
 
-.. cmd:: Hint @hint_definition : {+ @ident}
+.. _creating_hints:
 
-   The general command to add a hint to some databases :n:`{+ @ident}`.
+Creating Hints
+--------------
 
-   This command supports the :attr:`local`, :attr:`global` and :attr:`export`
-   locality attributes. When no locality is explictly given, the
+   The various `Hint` commands share these elements:
+
+   :n:`{? : {+ @ident } }` specifies the hint database(s) to add to.
+   *(Deprecated since version 8.10:* If no :token:`ident`\s
+   are given, the hint is added to the `core` database.)
+
+   Outside of sections, these commands support the :attr:`local`, :attr:`export`
+   and :attr:`global` attributes. :attr:`global` is the default.  Inside sections,
+   only the :attr:`local` attribute is supported because hints are local to sections.
+
+   When locality is specified, the
    command is :attr:`local` inside a section and :attr:`global` otherwise.
```

Comment:
This sentence is redundant with the previous paragraph. It can safely be removed.